build.sbt

```diff
@@ -49,23 +43,21 @@ val buildDebSettings = Seq(
   riffRaffArtifactResources += (file("cloudformation/membership-attribute-service.yaml") -> "cloudformation/membership-attribute-service.yaml"),
   Universal / javaOptions ++= Seq(
     "-Dpidfile.path=/dev/null",
-    "-J-XX:MaxRAMFraction=2",
-    "-J-XX:InitialRAMFraction=2",
+    "-J-XX:MaxRAMPercentage=50",
+    "-J-XX:InitialRAMPercentage=50",
     "-J-XX:MaxMetaspaceSize=500m",
     "-J-XX:+PrintGCDetails",
-    "-J-XX:+PrintGCDateStamps",
-    s"-J-Xloggc:/var/log/${packageName.value}/gc.log",
+    s"-J-Xlog:gc:/var/log/${packageName.value}/gc.log",
   ),
 )
 
 val `membership-common` =
   (project in file("membership-common"))
-    .enablePlugins(DynamoDBLocalPlugin, SbtPgp, Sonatype)
     .settings(
       Seq(
         name := "membership-common",
         organization := "com.gu",
-        scalaVersion := "2.13.10",
+        scalaVersion := "2.13.12",
```

Comment:
latest scala 2.13,  I tried separately bumping to scala 3.3 and although we were ok for libraries/binary compatibility with tiny tweaks, it needs our codebase bringing up to scratch first